<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualDisplayRow</class>
 <widget class="QWidget" name="VirtualDisplayRow">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="spacer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  <item>
   <widget class="QLabel" name="icon"/>
  </item>
  <item>
   <widget class="QLabel" name="labelId">
    <property name="text">
    <string>ID</string>
    </property>
   </widget>
  </item>
  <item>
   <widget class="QLabel" name="labelRes">
    <property name="text">
    <string>WxH</string>
    </property>
   </widget>
  </item>
   <item>
    <widget class="QPushButton" name="buttonRemove">
     <property name="toolTip">
      <string>Remove virtual display</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
